### ISSUE TITLE
Pundit::NotAuthorizedError: not allowed to new? this #<Band id: nil, name: "", created_at: nil, updated_at: nil, manager_id: ...

### ISSUE THREAD

**Issue by Shpigford** [Rockburg/rockburg#105]
https://sentry.io/share/issue/28a18f37482e462fa35d6c403fb2431d/

URL: `https://alpha.rockburg.com/bands/new`

```
Pundit::NotAuthorizedError: not allowed to new? this #<Band id: nil, name: "", created_at: nil, updated_at: nil, manager_id: nil, genre_id: nil, fans: 0, buzz: 0>
  app/controllers/bands_controller.rb:12:in `new'
    authorize(@band)
...
(63 additional frame(s) were not displayed)

Pundit::NotAuthorizedError: not allowed to new? this #<Band id: nil, name: "", created_at: nil, updated_at: nil, manager_id: nil, genre_id: nil, fans: 0, buzz: 0>
```

**Comment by nathanpalmer** [comment]
Does the full stack on Sentry include session information? I don't see much on the share. But what I'm guessing is this is from going to that URL when you are anonymous. But in either case we shouldn't expose that raw error to the user, we need to redirect with an error. 

I created PR #106 to address the redirect.

If there is more to that error let me know.

**Comment by Shpigford** [comment]
Fixed with #106
